### PR TITLE
MINOR: Clarify acceptable recovery lag config doc

### PR DESCRIPTION
**Pull request by vvcephei**
Clarify a misleading config description.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java [hunk 1]
```diff
@@ -329,8 +329,10 @@
 
     /** {@code acceptable.recovery.lag} */
     public static final String ACCEPTABLE_RECOVERY_LAG_CONFIG = "acceptable.recovery.lag";
-    private static final String ACCEPTABLE_RECOVERY_LAG_DOC = "The maximum acceptable lag (number of offsets to catch up) for a client to be considered caught-up for an active task." +
-                                                                  "Should correspond to a recovery time of well under a minute for a given workload. Must be at least 0.";
+    private static final String ACCEPTABLE_RECOVERY_LAG_DOC = "The maximum acceptable lag (number of offsets to catch up) for a client to be considered caught-up enough." +
```
Comment (ableegoldman) [inline streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java]:
```suggestion
    private static final String ACCEPTABLE_RECOVERY_LAG_DOC = "The maximum acceptable lag (number of offsets to catch up) for a client to be considered caught-up enough" +
```

**Review by ableegoldman — APPROVED** [review]
One extra `.` but otherwise LGTM

**Comment by ableegoldman** [comment]
Thanks for the update. I've had some users hit upon the same misunderstanding that I'm guessing sparked this

**Comment by vvcephei** [comment]
Thanks, @ableegoldman !

**Review by andrewchoi5 — APPROVED** [review]

**Pull request merged**